tests/cypress/tests/e2e/admin-dashboard.spec.js

```diff
@@ -0,0 +1,59 @@
+// element selectors
+const menuButton = '[data-testid="header-menu-dropdown-button"]';
+const menuOptionLogOut = '[data-testid="header-menu-option-log-out"]';
+const statesDropdown = '[data-testid="admin-dropdown"]';
+const enterProgramButton = '[data-testid="enter-program"]';
+
+before(() => {
+  // create a report as state user
+  cy.visit("/");
+  cy.authenticate("stateUser");
+  cy.findByRole("button", { name: "Enter MCPAR online" }).click();
+  cy.findAllByRole("button", { name: "Enter MCPAR online" }).click();
+  cy.findByRole("button", { name: "Add managed care program" }).click();
+
+  // fill out form fields
+  cy.findByLabelText("Program name").type("program title");
+  cy.get('input[name="reportingPeriodStartDate"]').type("07142023");
+  cy.get('input[name="reportingPeriodEndDate"]').type("07142026");
+  cy.findByRole("checkbox").focus().click();
+  cy.get("button[type=submit]").contains("Save").click();
+
+  // log out as state user
+  cy.get(menuButton).click();
+  cy.get(menuOptionLogOut).click();
+});
+
+beforeEach(() => {
+  // log in as admin
+  cy.visit("/");
+  cy.authenticate("adminUser");
+
+  // navigate to state report dashboard
+  cy.get(statesDropdown).select("MN");
+  cy.get("button[type=submit]").contains("Go to Report Dashboard").click();
+});
```

Comment:
There'll be some changes to these after the auth updates we put in last week. Let me know if you want to discuss them! Should simplify this in a way. Also we could consider Cucumber-ifying it - can get with @tblackwe on that if interested